packages/amplication-client/src/Resource/git/GitActions/RepositoryActions/GithubSyncDetails.tsx

```diff
@@ -34,8 +34,14 @@ function GithubSyncDetails({
     }).catch(console.error);
   }, [disconnectGitRepository, resourceWithRepository.id]);
   const errorMessage = formatError(disconnectErrorUpdate);
+  const gitProvider =
```

Comment:
it's a duplication, but we can live with that

Reply:
what is duplicated?

Reply:
the map of url

Reply:
So the comment is not in the right place. I fixed it. I mark this as resolved